### PR TITLE
feat: add ts solution to lc problem: No.935

### PR DESCRIPTION
**Pull request by iam-abhishek-yadav**

### CODE REVIEW

**Comment by iam-abhishek-yadav** [comment]
> Thanks for your contribution, you can propose changes to my suggestions, and if there are any doubts, we can start a discussion!

I can make the changes, but is there an particular reason to use const, if let is working fine. Is there any advantage, that I'm not aware of.

File: solution/0900-0999/0935.Knight Dialer/README.md [hunk 1]
```diff
@@ -224,6 +224,45 @@ public class Solution {
...
+        t[8] = f[1] + f[3];
+        t[9] = f[2] + f[4];
+
+        for (let i = 0; i < 10; ++i) {
+            f[i] = t[i] % MOD;
+        }
+    }
+
+    let ans: number = 0;
+    for (let v of f) {
```
[the remark this answers is not in the document]
Comment (iam-abhishek-yadav) [reply solution/0900-0999/0935.Knight Dialer/README.md]: updated

File: solution/0900-0999/0935.Knight Dialer/README.md [hunk 2]
```diff
@@ -224,6 +224,45 @@ public class Solution {
...
+    const MOD: number = 1e9 + 7;
+
+    if (n === 1) {
+        return 10;
+    }
+
+    let f: number[] = new Array(10).fill(1);
+
+    while (--n > 0) {
+        let t: number[] = new Array(10).fill(0);
```
[the remark this answers is not in the document]
Comment (iam-abhishek-yadav) [reply solution/0900-0999/0935.Knight Dialer/README.md]: updated

**Comment by thinkasany** [comment]
> > Thanks for your contribution, you can propose changes to my suggestions, and if there are any doubts, we can start a discussion!
> 
> I can make the changes, but is there an particular reason to use const, if let is working fine. Is there any advantage, that I'm not aware of.

`let`: for declaring mutable variables, `const`: for declaring immutable variables. 
At first glance it's easier to see where the references to variables are invariant or changeable, which makes it stricter, and if they're all `let`'s, it looks like they all change.
The `const` here means that the reference to t is immutable and cannot be reassigned, but it does not mean that the contents of the array are immutable.

**Review by thinkasany — APPROVED** [review]

**Pull request merged**